include/picongpu/plugins/PluginController.hpp

```diff
@@ -70,7 +70,7 @@
 #endif
 
 #if((ENABLE_OPENPMD == 1) && (openPMD_HAVE_HDF5 == 1))
-// needs to be openPMD-api only in the future to support adios
+// needs to be openPMD-api only in the future to support adios2
```

Comment:
Isn't it openPMD-api only by now? I think this comment is outdated and should instead mention why it only uses the HDF5 backend of openPMD.

Reply:
Correct. I think the message tried to express what you wrote, but did it not clearly. Rewrote